.github/workflows/check.yml

```diff
@@ -0,0 +1,90 @@
+name: Check
+on: 
+  pull_request:
+    branches: [ develop ]
+jobs:
+  pr-style:
+    name: pr-style
+    runs-on: ubuntu-latest
+    steps:
+      - uses: deepakputhraya/action-pr-title@master
+        with:
+          regex: '^\[[A-Z]+-\d+\]\s.+' # Regex the title should match.
+  code-style:
+    name: code-style
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install packages
+        run: |
+          sudo apt-get -y install build-essential texi2html
+          
+          # indent 2.2.11, the lastest version (2.2.12) make a different result
+          sudo apt-get -y install wget
+          wget https://ftp.gnu.org/gnu/indent/indent-2.2.11.tar.gz
+          tar xf indent-2.2.11.tar.gz
+          cd indent-2.2.11
+          ./configure
+          sudo make -j install
+          cd ..
+
+          # astyle
+          sudo apt-get -y install astyle
+      - uses: actions/checkout@v2
+      - id: files
+        uses: jitterbit/get-changed-files@v1
+      - name: Check code style
+        id: stylecheck
+        run: |
+          for f in ${{ steps.files.outputs.added_modified }}; do
+            ext=$(expr $f : ".*\(\..*\)")
+            case $ext in
+            .c|.h|.i)
+              indent -l120 -lc120 ${f}
+            ;;
+            .cpp|.hpp|.ipp)
+              astyle --style=gnu --mode=c --indent-namespaces --indent=spaces=2 -xT8 -xt4 --add-brackets --max-code-length=120 --align-pointer=name --indent-classes --pad-header --pad-first-paren-out ${f}
+            ;;
```

Comment:
Can we make a file for these lines and place it somewhere in the repository instead of embedding here?
For huge changes of files, a contributor may want to correct the indentation before creating PR.
So that we can guide our indentation rule just by giving a file link.

Reply:
It seems to make sense. We can put the indent script into _.github/workflows/_ like _.github/workflows/cppcheck-spr-list_. I'll try and let you know the result.

Reply:
@hgryoo I've separated indentation to _codestyle.sh_. Please check it.

Reply:
Looks good to me 👍